tests/integration/targets/aci_inventory_system/runme.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+for host in 173.36.219.68 173.36.219.69 173.36.219.70 ;
```

Comment:
Is it possible for the script to read the `inventory.networking` ini file to get the hosts? Otherwise we will need to remember to update this script if updating the hosts in the integration test inventory.

Reply:
I found this discussion online which may help?

```
You can use bash native parser to interpret ini values, by:

$ source <(grep = file.ini)

Sample file:

[section-a]
  var1=value1
  var2=value2
  IPS=( "1.2.3.4" "1.2.3.5" )

To access variables, you simply printing them: echo $var1. You may also use arrays as shown above (echo ${IPS[@]}).

If you only want a single value just grep for it:

source <(grep var1 file.ini)
```

Reply:
@akinross No worries, if its too much added complexity maybe we can simply add a comment to the inventory file so if we need to update it we shouldn't forget to update this script at the same time.

Reply:
@samiib, added new variable to the inventory file and made comments in the file. I am not a big fan of now having these IPs defined in this way but at least it is now in a single file. I think we should at a later stage look into cleaning and improving the structure of this file.

Reply:
@p3ck Thanks for the suggestion, decided to leverage it.